socorro/lib/search_common.py

```diff
@@ -310,6 +321,42 @@ def fix_hang_type_parameter(parameters):
                 hang_type.value = new_values
                 hang_type.data_type = 'int'
 
+    @staticmethod
+    def fix_version_parameter(parameters):
+        """Correct the version parameter.
+
+        If version is something finishing with a 'b' and operator is
+        'has terms', replace the operator with 'starts with' to query all
+        beta versions.
+        """
```

Comment:
Add a comment that explains that it's ok to use this for ALL products because we're using the rapid-beta process for all products we bother to support. (as discussed)